### DISCUSSION TITLE
Requesting Release given divergence of ENVs in master vs. Docker Hub

### DISCUSSION THREAD

**Discussion by excalq** [mongo-express/mongo-express#1620] [General]
Hello, and much gratitude to the maintainers of such a well-loved project.

We've spent a bit of time getting Mongo-Express working with AWS DocumentDB, with [several](https://github.com/mongo-express/mongo-express/issues/741) [red herrings](https://github.com/mongo-express/mongo-express/issues/484#issuecomment-515013150). Ultimately the subtle difference in ENV names (like `ME_CONFIG_MONGODB_CA_FILE` vs `ME_CONFIG_MONGODB_TLS_CA_FILE`) between what's shown on this repository's `master` README, and what's presented (missing) on [Docker Hub](https://hub.docker.com/_/mongo-express) was at issue. 

It was our mistake attempting to use `ME_CONFIG_MONGODB_TLS...` ENVs on an older image, to be sure. However, given the TLS changes, it'd be a wonderful development to have a 1.1 release made available on Docker Hub.

**Comment by rtritto** [comment]
The newest code is in the [ME repository](https://github.com/mongo-express/mongo-express) (master branch) and it can't be published under a release (version).
ME on NPM and ME as Docker Image on Docker Hub both use versioned releases of ME.
[MED repository](https://github.com/mongo-express/mongo-express-docker) is used to publish ME on Docker Hub but; the documentation (README) is outdated and the repository will be merged in the ME one (see #1705).
Basically each version (NPM and Docker Hub) uses related env vars listed in the README of the related version tag.
